pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -0,0 +1,58 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Interchain Tokens
+
+Axelar has [several ways](overview) to send [supported tokens](/resources/testnet#assets), but what if you want to send another ERC20 and have it be available on multiple chains? Axelar can help with our new Interchain Tokens!
+
```

Comment:
what makes an interchain token different from a standard ERC-20? You can send it cross-chain using one function call. It's standard ERC-20 interface + an extra function to support cross-chain capabilities.

Reply:
Interchain Tokens are standard ERC-20s that you can also send cross-chain. To send cross-chain, you have to approve the token and then send via the linker, so it's be two separate function calls from my understanding. 

I actually think we should build a `send` method into the remote chain ERC-20s, so you interact with the token rather than the token linker. This would let us approve and send all in one function call.

This paragraph was intended to say "what if you want to send tokens that `getDepositAddress` and `sendToken` don't support?"